### DISCUSSION TITLE
Ginger Source Control Connection

### DISCUSSION THREAD

**Discussion by iHarmonyInternal** [Ginger-Automation/Ginger#3366] [Q&A]
Hey Guys, 
I'm stuck at this point and I cannot understand where I went wrong. I have given the correct credentials, and even then it's showing this Error.
![image](https://github.com/Ginger-Automation/Ginger/assets/142913472/342ad8d0-a1f3-4578-90e1-8d5d100c2dd5)

I would appreciate it if this issue is solved as soon as possible.

**Comment by ShevOleks** [comment]
Hellow @iHarmonyInternal !
Did you solve this problem?

**Comment by iHarmonyInternal** [comment]
Not yet, Oleksandr!

it would be a great help if you can help us!!
